k8s/upgrades/0.8.2-0.9.0/jiva/jiva_upgrade_runtask.yaml

```diff
@@ -1,43 +1,116 @@
-# Runtask for upgrading a jiva volume
+# Sample Runtask for upgrading a jiva volume
 
 # CASTemplate jiva-volume-update-0.8.2-0.9.0 is
 # used to upgrade a jiva volume
-# TODO: update the file with proper naming conventions and error messages
-# TODO: Test need to be performed on multi-node cluster
 apiVersion: openebs.io/v1alpha1
 kind: CASTemplate
 metadata:
   name: jiva-volume-update-0.8.2-0.9.0
 spec:
   defaultConfig:
-  - name: baseversion
+  # Base version is the version from which upgrade can happen.
+  # This CAS template is does not support  upgrade of OpenEBS version
+  # whose version is anything other that specified base version
+  # Using this CAS templates, once can upgrade from OpenEBS version
+  # 0.8.2 to 0.9.0 only.
+  - name: baseVersion
     value: "0.8.2"
-  - name: targetversion
+  - name: targetVersion
```

Comment:
can we put exact docker images?
Do we need to mention `-ci`

Reply:
Currently 0.9.0 tag is not available so went with ci image tag

Reply:
@AmitKumarDas can we update it once 0.9.0 images are available